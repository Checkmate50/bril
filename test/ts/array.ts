var arr = [3, 2, 1];
arr[2] = 5;
console.log(arr[1]);