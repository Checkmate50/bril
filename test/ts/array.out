main {
  v0: int[3] = new int[3];
  v1: int = const 0;
  v2: int = const 3;
  set v0 v1 v2;
  v3: int = const 1;
  v4: int = const 2;
  set v0 v3 v4;
  v5: int = const 2;
  v6: int = const 1;
  set v0 v5 v6;
  arr: int[0] = id v0;
  v7: int = const 5;
  v8: int = const 2;
  v9: int[0] = id arr;
  set v9 v8 v7;
  v10: int = const 0;
  v11: int = const 1;
  v12: int[0] = id arr;
  v13: int = index v12 v11;
  print v13;
  v14: int = const 0;
}
